docs/projects/fps/tutorial.md

```diff
@@ -141,12 +142,7 @@ For health packs we will do the latter, so that when the game begins there will
 
 #### Editing snapshot generation
 
-The SpatialOS menu option in the Unity editor include an item **"Generate FPS Snapshot"**. This runs the script `Assets/Fps/Scripts/Editor/SnapshotGenerator/SnapshotMenu.cs`, which you can find from within your Unity editor.
-
-We will modify the snapshot generating logic in two steps:
-
-1. Adding a function to create and add a `HealthPack` entity.
-2. Fix the package import settings so use of `Vector3f` is valid.
+The SpatialOS menu option in the Unity editor include an item **"Generate FPS Snapshot"**. This runs the script `Assets/Fps/Scripts/Editor/SnapshotGenerator/SnapshotMenu.cs`, which you can find from within your Unity editor. We will now modify the snapshot generating logic to add a `HealthPack` entity to out snapshot.
```

Comment:
"snapshot generating" to "snapshot generation"

Reply:
Fixed!